static/app/components/mutedBox.tsx

```diff
@@ -11,10 +9,10 @@ type Props = {
   statusDetails: ResolutionStatusDetails;
 };
 
-class MutedBox extends PureComponent<Props> {
```

Comment:
Should we use `React.memo`?

Reply:
We could, I don't think this component would re-render very often, nor are its render passes expensive.